prov/gni/src/gnix_rma.c

```diff
@@ -158,17 +158,25 @@ static void __gnix_rma_copy_chained_get_data(struct gnix_fab_req *req)
 		GNIX_INFO(FI_LOG_EP_DATA, "writing %d bytes to %p\n",
 			  tail_len, addr);
 		memcpy((void *)addr,
-		       (void *) ((uint8_t *) req->int_tx_buf + GNI_READ_ALIGN),
-		       tail_len);
+			   (void *) ((uint8_t *) req->int_tx_buf + GNI_READ_ALIGN),
+			   tail_len);
 	}
 }
 
 static void __gnix_rma_more_fr_complete(struct gnix_fab_req *req)
 {
+	int rc;
+
 	if (req->flags & FI_LOCAL_MR) {
 		GNIX_INFO(FI_LOG_EP_DATA, "freeing auto-reg MR: %p\n",
 			  req->rma.loc_md);
-		fi_close(&req->rma.loc_md->mr_fid.fid);
+		rc = fi_close(&req->rma.loc_md->mr_fid.fid);
+		if (rc != FI_SUCCESS)
+			GNIX_FATAL(FI_LOG_DOMAIN,
+				"failed to close auto-registration, "
```

Comment:
At least in githubs system the "failed" falls under the AL of FATAL. If it under the FI_LOG_DOMAIN in the regular code this is fine and the comment can be ignored